docs/customize/administration/administration_interface.md

```diff
@@ -0,0 +1,346 @@
+# Administration panel customisation
+
+This guide will show how to create and customize administration views.
+
+## Intended audience
+
+This guide is intended for developers of InvenioRDM.
```

Comment:
No other section in the customize has explicit intended audience. However, if you want to add it in the _customize_ section it should be sysadmins not developers.